packages/components/src/ConnectionCard/TripSector.js

```diff
@@ -56,10 +38,14 @@ export default function TripSector({
         </View>
         <View style={styles.tripItems}>
           <View style={styles.time}>
-            <Text numberOfLines={1}>{departureTime}</Text>
-            <Text numberOfLines={1}>{arrivalTime}</Text>
+            <Text style={styles.highlightedText} numberOfLines={1}>
+              {departureTime}
+            </Text>
+            <Text style={styles.highlightedText} numberOfLines={1}>
+              {arrivalTime}
+            </Text>
           </View>
-          <TimelineArrow />
+          <Timeline />
```

Comment:
I am not sure about the naming. I think `Timeline` is too generic, when `TimelineArrow` was at least a bit more descriptive.